middleware/logging/writer.go

```diff
@@ -43,3 +43,9 @@ func (w *responseWriter) Write(b []byte) (int, error) {
 	n, err := w.bodyWriter.Write(b)
 	return n, fault.Wrap(err)
 }
+
+// Flush implements the http.Flusher interface. This is needed to implement server-sent events.
```

Comment:
It may be needed but doesn't tell why :)

Reply:
True, I've added the reason